.github/workflows/tests.yml

```diff
@@ -30,9 +30,31 @@ jobs:
         run: make install
       - name: Run linter
         run: make lint
+  openapi:
+    strategy:
+      fail-fast: false
+      matrix:
```

Comment:
Why the matrix if there is only 1 combination?

Reply:
removed in https://github.com/canonical/cos-registration-server/pull/49/commits/284fbcf8b4d7ff68225e43565062bc97288edd4c